### ISSUE TITLE
Instructions on how to contribute? [Question] - 

### ISSUE THREAD

**Issue by CrocWorks** [Monika-After-Story/MonikaModDev#1721]
Are some of the instructions on https://github.com/Monika-After-Story/MonikaModDev/blob/master/CONTRIBUTING.md outdated?

I read somewhere that new Monika sprite codes were added, and looking at some of the other submissions the sprite codes are different from the ones listed in https://github.com/Monika-After-Story/MonikaModDev/blob/master/docs/MonikaCheatsheetupdate2.jpg

Also, what is the difference between Issues and Pull requests?

Finally, if I were to contribute something, would I just attach a notepad file of code to a suggestion like this one? It says also says in the instructions to test bug fixes before submitting; does this also apply to adding events? I'd pretty much just copy the syntax of another event and modify labels and text, so would an official debugging still be necessary if a template were followed?

Just curious in case I might want to contribute some writing eventually (I have some minor coding experience and could probably write text okay, but I have no real experience with using Github).

**Comment by jmwall24** [comment]
The new expressions/sprites guidelines are here https://github.com/Monika-After-Story/MonikaModDev/blob/master/Monika%20After%20Story/game/sprite-chart.rpy

To contribute something like dialogue, go on that Contribution page you linked to and go down to 'Your First Contribution.'  Everything from there to the bottom is how to contribute to the mod.

**Comment by Rai99** [comment]
If you're writing something completely new, put it in the content branch. 
If you're editing something that already exists, put it in the enhancement branch. 

There are new sprite codes but there isn't any cheatsheet for the new ones yet. 

To put a contribution in, go to the code section just to the left of issues and click on the branch:master to select the branch you want. Go into Monika After Story > Games and select the document you want to add to (topics, moods, greetings, etc). Click the pencil icon to the right of the  Raw/Blame/History buttons to edit the document. Scroll around and pick a spot to put your contribution but ideally don't cut somebody else's in half. Topics usually end with "return" so generally put it somewhere under one of those. Scroll to the bottom of the page (not the document) and add a title and description and press propose file change. The rest should be simple.

**Comment by Rai99** [comment]
Or yeah just do what jmwall24 says since I don't even have the fabled contributor tag yet.

**Comment by jmwall24** [comment]
lol that contributor tag is automatic once your topic is merged into the content branch and then the content branch with your work in it is merged to the master branch.  It just takes time for the team to review everything, you're a good writer you'll have it soon enough.

**Comment by CrocWorks** [comment]
Cool, thanks. I think I figured out where I would add stuff. Should I post what I want to add in a new Issues topic first to get feedback and then add code, or is it better to drop it straight into the code?

**Comment by Rai99** [comment]
I usually put up an issue first to make sure I haven't lost it and I'm not about to write a garbage contribution.

**Comment by CrocWorks** [comment]
Is just a wordpad file good enough?

**Comment by Rai99** [comment]
Should be. Anything that holds text for you. You could even completely write the entire thing in the github document itself although probably not the best idea.

**Comment by Rai99** [comment]
And thanks. I was never too sure if my writing was good enough for MAS and always wondered if people actually liked my ideas.

**Comment by CrocWorks** [comment]
Alright, thanks for clarifying this for me. When I have something new, I'll make a new issue for it.
